lib/LocalServer.js

```diff
@@ -57,14 +57,14 @@ LocalServer.prototype.getConnectionUrl = function() {
     return this.tunneledUrl || this.externalServerUrl + ":" + this.port;
 };
 
-LocalServer.prototype.reset = function() {
+LocalServer.prototype.reset = function(savePath) {
```

Comment:
I think Server should not know about this particular parameter, I think we should make configuration instance more accessible (singleton or similar) so you don't need to pass all params all way down. Alternative approach is to pass required params to `specReporters.initialize (config or savePath)` somewhere in paramedic initialization.